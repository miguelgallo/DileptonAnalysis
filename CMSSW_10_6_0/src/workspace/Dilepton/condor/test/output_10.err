Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
ImportError: No module named site
WARNING: In non-interactive mode release checks e.g. deprecated releases, production architectures are disabled.
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCand
Error in <TTree::SetBranchAddress>: unknown branch -> nGenMuonCandOutOfAccept
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenMuonCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> nGenProtCand
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_e
Error in <TTree::SetBranchAddress>: unknown branch -> GenProtCand_status
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_mass
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_pt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_eta
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_phi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dpt
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_dphi
Error in <TTree::SetBranchAddress>: unknown branch -> GenPair_3Dangle

 *** Break *** segmentation violation



===========================================================
There was a crash.
This is the entire stack trace of all threads:
===========================================================
#0  0x00002b891d64141c in waitpid () from /lib64/libc.so.6
#1  0x00002b891d5bef12 in do_system () from /lib64/libc.so.6
#2  0x00002b891cf1752d in TUnixSystem::StackTrace() () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#3  0x00002b891cf19dd4 in TUnixSystem::DispatchSignals(ESignals) () from /cvmfs/cms.cern.ch/slc7_amd64_gcc700/lcg/root/6.14.09-pafccj/lib/libCore.so
#4  <signal handler called>
#5  0x00002b892a12fe5c in ?? ()
#6  0x3fba9999a0000000 in ?? ()
#7  0x3fb2c51ec0000000 in ?? ()
#8  0x3fa2d999a0000000 in ?? ()
#9  0xbfb0147ae0000000 in ?? ()
#10 0x3fd04a3d80000000 in ?? ()
#11 0x3f7b3851e0000000 in ?? ()
#12 0x3fbceb8520000000 in ?? ()
#13 0x3fbb1eb860000000 in ?? ()
#14 0xbfd0947ae0000000 in ?? ()
#15 0xbf91cf5c20000000 in ?? ()
#16 0xbfa2f851e0000000 in ?? ()
#17 0xbfa24a3d80000000 in ?? ()
#18 0x3fbc051ec0000000 in ?? ()
#19 0x3f9f28f5c0000000 in ?? ()
#20 0xbfa6800000000000 in ?? ()
#21 0x3fa1fd70a0000000 in ?? ()
#22 0x3fa28cccc0000000 in ?? ()
#23 0x3fbd0a3d80000000 in ?? ()
#24 0xbfd3547ae0000000 in ?? ()
#25 0x3fa07d70a0000000 in ?? ()
#26 0xbfd2970a40000000 in ?? ()
#27 0x3fbaa3d700000000 in ?? ()
#28 0xbf54eb8520000000 in ?? ()
#29 0x3fc58f5c20000000 in ?? ()
#30 0xbfb3547ae0000000 in ?? ()
#31 0x3fb6d1eb80000000 in ?? ()
#32 0xbf45fae140000000 in ?? ()
#33 0x3fd25c2900000000 in ?? ()
#34 0xbfb6ae1480000000 in ?? ()
#35 0xbfd00f5c20000000 in ?? ()
#36 0x3faf051ec0000000 in ?? ()
#37 0x3f95147ae0000000 in ?? ()
#38 0x3fd1a3d700000000 in ?? ()
#39 0x3fbae66660000000 in ?? ()
#40 0xbfd2d70a40000000 in ?? ()
#41 0x3fbb428f60000000 in ?? ()
#42 0x3fc50a3d80000000 in ?? ()
#43 0x3fd251eb80000000 in ?? ()
#44 0x3fc21eb860000000 in ?? ()
#45 0xbfe847ae20000000 in ?? ()
#46 0xbfc94cccc0000000 in ?? ()
#47 0x0000000000000000 in ?? ()
===========================================================


The lines below might hint at the cause of the crash.
You may get help by asking at the ROOT forum http://root.cern.ch/forum
Only if you are really convinced it is a bug in ROOT then please submit a
report at http://root.cern.ch/bugs Please post the ENTIRE stack trace
from above as an attachment in addition to anything else
that might help us fixing this issue.
===========================================================
#5  0x00002b892a12fe5c in ?? ()
#6  0x3fba9999a0000000 in ?? ()
#7  0x3fb2c51ec0000000 in ?? ()
#8  0x3fa2d999a0000000 in ?? ()
#9  0xbfb0147ae0000000 in ?? ()
#10 0x3fd04a3d80000000 in ?? ()
#11 0x3f7b3851e0000000 in ?? ()
#12 0x3fbceb8520000000 in ?? ()
#13 0x3fbb1eb860000000 in ?? ()
#14 0xbfd0947ae0000000 in ?? ()
#15 0xbf91cf5c20000000 in ?? ()
#16 0xbfa2f851e0000000 in ?? ()
#17 0xbfa24a3d80000000 in ?? ()
#18 0x3fbc051ec0000000 in ?? ()
#19 0x3f9f28f5c0000000 in ?? ()
#20 0xbfa6800000000000 in ?? ()
#21 0x3fa1fd70a0000000 in ?? ()
#22 0x3fa28cccc0000000 in ?? ()
#23 0x3fbd0a3d80000000 in ?? ()
#24 0xbfd3547ae0000000 in ?? ()
#25 0x3fa07d70a0000000 in ?? ()
#26 0xbfd2970a40000000 in ?? ()
#27 0x3fbaa3d700000000 in ?? ()
#28 0xbf54eb8520000000 in ?? ()
#29 0x3fc58f5c20000000 in ?? ()
#30 0xbfb3547ae0000000 in ?? ()
#31 0x3fb6d1eb80000000 in ?? ()
#32 0xbf45fae140000000 in ?? ()
#33 0x3fd25c2900000000 in ?? ()
#34 0xbfb6ae1480000000 in ?? ()
#35 0xbfd00f5c20000000 in ?? ()
#36 0x3faf051ec0000000 in ?? ()
#37 0x3f95147ae0000000 in ?? ()
#38 0x3fd1a3d700000000 in ?? ()
#39 0x3fbae66660000000 in ?? ()
#40 0xbfd2d70a40000000 in ?? ()
#41 0x3fbb428f60000000 in ?? ()
#42 0x3fc50a3d80000000 in ?? ()
#43 0x3fd251eb80000000 in ?? ()
#44 0x3fc21eb860000000 in ?? ()
#45 0xbfe847ae20000000 in ?? ()
#46 0xbfc94cccc0000000 in ?? ()
#47 0x0000000000000000 in ?? ()
===========================================================


