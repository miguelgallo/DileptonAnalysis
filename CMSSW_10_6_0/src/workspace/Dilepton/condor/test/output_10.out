copying file
running
0cortes: 3491988
1cortes: 49700
2cortes: 46160
3cortes: 44468
4cortes: 20741
5cortes: 20741
6cortes: 12482
7cortes: 1715
8cortes: 27
moving file
